vendor/github.com/stripe/goproxy/https.go

```diff
@@ -543,11 +544,19 @@ func (proxy *ProxyHttpServer) connectDialProxyWithContext(ctx *ProxyCtx, proxyHo
 		c = tls.Client(c, proxy.Tr.TLSClientConfig)
 	}
 
+	hdr := make(http.Header)
```

Comment:
I'd make this a little more explicit - something like `proxyAuthHeader`

Reply:
yep changed!